pkg/constants/constants.go

```diff
@@ -333,4 +333,7 @@ const (
 	MainnetCChainICMRegistryAddress = "0x7C43605E14F391720e1b37E49C78C4b03A488d98"
 	FujiCChainICMRegistryAddress    = "0xF86Cb19Ad8405AEFa7d09C778215D2Cb6eBfB228"
 	ValidatorUptimeDeductible       = uint64(10) // seconds to make sure all L1 validators would agree on uptime
+
+	// Aggregator
+	DefaultAggregatorLogLevel = "Debug"
```

Comment:
default value should be `Info` not `Debug`. User should explicitly set it to Debug if needed

Reply:
Debug is a better default IMO as Info usually does not provide hints into the error cause if any is obtained.
This is anyway going into a log file so not cluttering the user ux. Take into account that we want to 
know the cause for an aggregator failure just after seeing an error, and avoid trying to duplicate the issue
(if that is possible at all).